src/chopper_layout.cpp

```diff
@@ -142,6 +142,8 @@ size_t determine_best_number_of_technical_bins(chopper::layout::data_store & dat
 
     size_t const total_kmer_count = std::accumulate(data.kmer_counts.begin(), data.kmer_counts.end(), size_t{});
 
+    double t_max_64_memory = -1.0;
```

Comment:
```suggestion
    size_t t_max_64_memory{};
```

Wouldn't that work? if `get_total_hibf_memory_size` would return `0`, something has to be really wrong, no?